src/wallet-parity.js

```diff
@@ -52,7 +52,8 @@ function createParityWalletMiddleware({ walletMiddleware }){
     })
   }
 
-  async function checkRequest (req, res) {
+  async function checkRequest (_req, res) {
+    // eslint-disable-next-line no-undef
     const result = parityRequests[reqId]
```

Comment:
Even though I disabled this `no-undef` error, the `reqId` variable in this line does not have a definition or value, and also I was unable to track down any usage of `createParityWalletMiddleware`.

Reply:
On review, we should delete this file and all references to it. It is not referenced anywhere in the extension codebase, and clearly it's in a bad state anyway.

Reply:
I don't think this file is referenced anywhere in this repo either. I did check the extension and could not find any reference there either. Should I remove this file as a part of this PR or a separate PR??

Reply:
I wanted to say just do it here, but we should be good and do it in a separate PR, as I've done here: https://github.com/MetaMask/eth-json-rpc-middleware/pull/63